psi4/driver/schema_wrapper.py

```diff
@@ -185,10 +214,60 @@ def _read_output(outfile):
 
         return output
     except OSError:
-        return "Could not read output file"
+        return "Could not read output file."
+
+
+def _quiet_remove(filename):
+    """
+    Destroy the created at file at exit, pass if error.
+    """
+    try:
+        os.unlink(filename)
+    except OSError:
+        pass
+
+
+def run_qcschema(input_data, clean=True):
```

Comment:
This is just a remark: The function names are a little confusing as now we have both `run_json_qcschema` and `run_qcschema`.

Reply:
`run_json/run_qcschma` are the top level functions where `run_json` will be deprecated.

Reply:
I got a little tangled on the fn names, too. might move to `_run_json_qcschema` in future.